<launch>
  <!-- <arg name="port" default="ttyACM0" /> -->

  <node name="usb_cam_lane" pkg="usb_cam" type="usb_cam_node" output="screen" >
    <param name="video_device" value="/dev/video4" />
    <param name="image_width" value="1280" />
    <param name="image_height" value="720" />
    <param name="pixel_format" value="mjpeg" />
    <param name="camera_frame_id" value="usb_cam" />
    <param name="io_method" value="mmap"/>
  </node>
  <node name="usb_cam_light" pkg="usb_cam" type="usb_cam_node" output="screen" >
    <param name="video_device" value="/dev/video2" />
    <param name="image_width" value="1280" />
    <param name="image_height" value="720" />
    <param name="pixel_format" value="mjpeg" />
    <param name="camera_frame_id" value="usb_cam" />
    <param name="io_method" value="mmap"/>
  </node>

  <node name="arduino" pkg="rosserial_python" type="serial_node.py" output="screen" >
    <param name="port" value="$(arg port)" />
    <param name="baud" value="115200" />
  </node>
  
  <include file="$(find rplidar_ros)/launch/rplidar_a1.launch" />
  
  <node name="rviz" pkg="rviz" type="rviz" args="-d $(find rplidar_ros)/rviz/rplidar.rviz" />
</launch>